packages/client/src/hooks/useNFTs.js

```diff
@@ -100,6 +139,16 @@ export default function useNFTs() {
     }).catch(console.error);
 
     const collections = (identifiers && identifiers[1]) ?? [];
+
+    if (collections.length === 0) {
+      dispatch({
+        type: "SET_NFTS",
+        payload: {
+          [treasuryAddr]: {},
+        },
+      });
```

Comment:
Forgot to return here?

Reply:
Yup. But even if no return is provided, the for loop will be skipped since the collection is empty